java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -485,7 +486,7 @@ private static Object[] scriptingAndFunctionsCommands(BaseTransaction<?> transac
                 };
 
         transaction
-                .customCommand(new String[] {"function", "flush", "sync"})
```

Comment:
should we test functionFlush without the FlushMode argument or is it not necessary?

Reply:
Added  to the end, thank you.